packages/office-addin-manifest/test/test.ts

```diff
@@ -623,27 +624,56 @@ describe("Unit Tests", function() {
       });
     });
   });
+  describe("export.ts", function() {
+    describe("exportMetadataPackage()", function() {
+      it("export manifest", async function() {
+        this.timeout(6000);
+        const testFolder = path.resolve("./testExecution");
+        const manifestPath = path.normalize("test/manifests/manifest.json");
+        const assetsPath = path.normalize("test/assets");
+        const outputPath = path.normalize(`${testFolder}/testPackage.zip`);
+        const outputFile = await exportMetadataPackage(outputPath, manifestPath, assetsPath);
+
+        assert.strictEqual(outputFile, outputPath, "Output path should match the argument");
+        assert.strictEqual(fs.existsSync(outputFile), true, "Output file should exist");
+        
+        // Cleanup
+        fs.rmSync(testFolder, { recursive: true });
+      });
+      it("invalid manifest path", async function() {
+        this.timeout(6000);
+        let result: string = "";
+        const invalidManifestPath = path.normalize(`${manifestTestFolder}/foo/manifest.json`);
+        try {
+          await exportMetadataPackage(invalidManifestPath);
+        } catch (err: any) {
+          result = err.message;
+        }
+        assert.strictEqual(result.indexOf("The file 'manifest.json' does not exist") >= 0, true);
```

Comment:
Can we just test if the error exists? 
Maybe we will change the message in the future and we don't want for the tests to fail.

Reply:
If we return different errors we wouldn't be figuring out if the right one was the one received.